prboom2/src/hu_stuff.c

```diff
@@ -1249,6 +1249,16 @@ int HU_GetHealthColor(int health, int def)
 
 int HU_GetArmorColor(int armor, int def)
 {
+// [pull req] suggested change: armor color dictated by type (Advanced HUD)
```

Comment:
Once we merge your PR, these comments will not make any sense anymore. Please remove phrases like "pull req" and "suggested changes".

Reply:
Understood! Apologies for including those phrases in the comments, they were basically just for myself. They have been removed in the latest commit.